### ISSUE TITLE
O365 integration not detectind drive business

### ISSUE THREAD

**Issue by rachidSabah** [microsoft/o365-moodle#2250]
Hello everyone 

Anyone can help I am getting the following message when configuring the openid connect to detect o365 business drive:

'We could not determine your OneDrive for Business [URL](https://moodle.org/mod/glossary/showentry.php?eid=31&displayformat=dictionary).<br />Please make sure "Microsoft 365 SharePoint Online" has been added to your registered application in Azure AD

**Comment by nenorojas** [comment]
Hi,

Have you used the PowerShell script location inside local_o365 to create the application in Azure? 
/local/o365/scripts/Moodle-AzureAD-Powershell.zip

That should create an application will the permission you need to allow detection of that URL after you click in the button in Moodle to "Provide Admin Consent".

If that doesn't work, you can enter the URL manually and save. 

Regards

Oswaldo

**Comment by rachidSabah** [comment]
> Hi,
Yes I used the powershell when I provide Admin Constent but on o365 when I push detect it not detecting it. Further onedrive it shows within the moodle repository as well I am able to embed video directly from drive despite the video only visible to a specific users with microsoft account, external won't be able to see the video unless I designated them as sharing profile. No matter even I choose on my o365 sharepoint that videos file on drive is visible to anyone with a link.
I don't know why these restrictions.
Google Drive works fine without any issues
